recipes/AffjaxPostNode/src/Main.purs

```diff
@@ -0,0 +1,51 @@
+module AffjaxPostNode.Main where
+
+import Data.Argonaut
+
+import Affjax as AX
+import Affjax.RequestBody (json)
+import Affjax.ResponseFormat as ResponseFormat
+import Data.Argonaut as J
+import Data.Either (Either(..))
+import Data.Maybe (Maybe(..))
+import Effect (Effect)
+import Effect.Aff (launchAff_)
+import Effect.Class.Console (log)
+import Prelude (Unit, bind, ($), (<>))
+
+type Post = { id     :: Maybe Int
+            , title  :: String
+            , body   :: String
+            , userId :: Int
+            }
+
+postToJson :: Post -> Json
+postToJson = encodeJson
+
+postFromJson :: Json -> Either JsonDecodeError Post
+postFromJson = decodeJson
+
+main :: Effect Unit
+main = launchAff_ $ do
+  let 
+    postdata :: Post
+    postdata =  { id    : Nothing -- note how this returns filled-in if service works
+                , title : "title"
+                , body  : "body"
+                , userId: 22 }
+
+    endpoint :: String
+    endpoint = "http://jsonplaceholder.typicode.com/posts"
+    
+  result <- AX.post ResponseFormat.json endpoint (Just $ json $ encodeJson postdata)
+  case result of
+
+    Left err       -> log $ "POST " <> endpoint <> " response failed to decode: " 
+      <> AX.printError err
+
+    Right response -> log $ "POST " <> endpoint <> " response: "
+      <> J.stringify response.body
```

Comment:
Similarly here. Perhaps this should be:
```purescript
Right response -> do
  log $ "POST " <> endpoint <> " response: " <> J.stringify response.body
  log $ ">>> " <> show (jsonToPost response.body)
```

Reply:
d'you mean:
```haskell
    Right response -> do
      log $ "POST " <> endpoint <> " response: " <> show (postFromJson response.body)
```
??

we don't need to show the `J.stringify` i think?

Reply:
No, I really did mean what I wrote above. The first line will show the JSON in a string format. The second line will convert the body into our data type and then use that type's `Show` instance to convert it into a String. 

By having both, it shows the JSON you got back and the resulting data type you produced by parsing the JSON via `decodeJson`